Week1/homework/app.js

```diff
@@ -1,11 +1,165 @@
+/* eslint-disable camelcase */
+
 'use strict';
 
 {
+  // #region 1.1_&_1.2
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'The 7 Habits of Highly Effective People',
+    'Animal Farm',
+    'Its Your Ship',
+    'How to Win Friends and Influence People',
+    'Good to Great',
+    'Sapiens',
+    'Little Prince',
+    'The Alchemist',
+    'The Art of War',
+    'Elfabe',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  /*
+  bookTitles.forEach(item => {
+    console.log(item);
+  });
+  */
+  // #endregion 1.1_&_1.2
+
+  // #region 1.3
+  function idNameConvention(name) {
+    const reg = /\w+/gi;
```

Comment:
I do like the creative use of regex to turn a normal title into a valid id!

Reply:
Thanks.